src/crsAdapter/TomaAdapter.js

```diff
@@ -488,6 +527,25 @@ class TomaAdapter {
 
     /**
      * @private
+     * @param service object
+     * @param xml object
+     * @param lineNumber number
+     */
+    assignRoundTripServiceFromAdapterObjectToXmlObject(service, xml, lineNumber) {
+        xml['KindOfService.' + lineNumber] = CONFIG.crs.serviceTypes.roundTrip;
+        xml['ServiceCode.' + lineNumber] = service.bookingId;
+        xml['Accommodation.' + lineNumber] = service.destination;
+        xml['Count.' + lineNumber] = service.no;
+        xml['From.' + lineNumber] = moment(service.startDate, this.options.useDateFormat).format(CONFIG.crs.dateFormat);
+        xml['To.' + lineNumber] = moment(service.endDate, this.options.useDateFormat).format(CONFIG.crs.dateFormat);
+        xml['Title.' + lineNumber] = service.title;
+        xml['Name.' + lineNumber] = service.name;
+        xml['Reduction.' + lineNumber] = service.birthday ? service.birthday : service.age;
```

Comment:
you can use a shorter version: `service.birthday || service.age` it is the same like in PHP the `service.birthday ?: service.age`